sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_auth_events_v1/query.sql

```diff
@@ -10,7 +10,7 @@ WITH base AS (
           jsonPayload.* REPLACE (
             (
               SELECT AS STRUCT
-                jsonPayload.fields.* EXCEPT (device_id, user_id) REPLACE(
+                jsonPayload.fields.* EXCEPT (user_id) REPLACE(
                   -- See https://bugzilla.mozilla.org/show_bug.cgi?id=1707571
```

Comment:
add `TO_HEX(SHA256(jsonPayload.fields.device_id)) AS device_id,` after `user_id` 8 lines below and:
```suggestion
                jsonPayload.fields.* EXCEPT (device_id, user_id) REPLACE(
                  -- See https://bugzilla.mozilla.org/show_bug.cgi?id=1707571
```